app/controllers/catalog.ts

```diff
@@ -103,4 +98,14 @@ export default class CatalogController extends Controller {
       .filterBy('featureIsProductWalkthrough')
       .rejectBy('isDismissed')[0] as FeatureSuggestionModel | null;
   }
+
+  shouldDisplayCourse(course: CourseModel) {
+    const isAlphaOrDeprecated = course.releaseStatusIsAlpha || course.releaseStatusIsDeprecated;
+    const isAuthorizedUser =
```

Comment:
```suggestion
    const userIsStaffOrCourseAuthor =
```

Reply:
(don't introduce "new" names unless they actually simplify logic or compress things, sticking to existing names/concepts keeps things simpler)

Reply:
On second thought, let's change this behaviour - for "alpha" it makes sense to keep a course in the list for authors / staff since they'd need to test a course actively. For "deprecated" this isn't the case, you'd hardly every want to visit a deprecated course. Let's not show deprecated courses even to authors/staff, they can always navigate to the URL (/courses/docker) directly if needed